src/EventGrid/EventGrid/Az.EventGrid.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.4.0'
+ModuleVersion = '1.5.0'
```

Comment:
No need to update the version manually. Our tools will do it when build the release.

Reply:
Removed manual version increment